articles/extensions/delegated-admin/v2/index.md

```diff
@@ -113,7 +133,7 @@ This rule gives users from the `IT Department` the `Delegated Admin - Administra
 ```js
 function (user, context, callback) {
  if (context.clientID === 'CLIENT_ID') {
-   const namespace = 'https://${account.tenant}.us8.webtask.io/auth0-delegated-admin';
+   const namespace = 'https://${account.tenant}.prod-us-node12.webtask.io/auth0-delegated-admin';
```

Comment:
I think this may be another case where we need to provide two different URLs based on the version of Node they are running.